travel_platform/izi_travel_client.py

```diff
@@ -1,6 +1,8 @@
+import os
```

Comment:
I suggest to rename all appropriate files into `izi_travel_` instead of `izitravel_`.